Source/WebCore/html/HTMLLinkElement.cpp

```diff
@@ -202,6 +205,10 @@ void HTMLLinkElement::attributeChanged(const QualifiedName& name, const AtomStri
             m_relList->associatedAttributeValueChanged();
         if (didMutateRel)
             process();
+#if ENABLE(WEB_PAGE_SPATIAL_BACKDROP)
+        if (equalLettersIgnoringASCIICase(oldValue, "spatial-backdrop"_s) && !equalLettersIgnoringASCIICase(newValue, "spatial-backdrop"_s))
+            document().spatialBackdropLinkElementChanged();
+#endif
```

Comment:
This seems bad. `m_relAttribute` and `parsedRel` already store the relevant booleans for this.

This also wouldn't catch mutations that differ by whitespace.

Reply:
It also seems like you end up invoking this multiple times in certain cases. Once through `process()` on the line before and once through this?